deploy/docs/Working_with_container_registries.md

```diff
@@ -3,41 +3,59 @@
 ## Authenticating with container registry
 
 Sumo Logic container images used for collection are currently hosted on
-[Docker Hub](https://hub.docker.com/) which
-[requires authentication to provide a higher quota for image pulls][docker-rate-limit].
+[Amazon Public ECR](https://gallery.ecr.aws/) which
+[requires authentication to provide a higher quota for image pulls][aws-public-ecr-docs].
 
-In order to authenticate with the container registry hosted on
-[Docker Hub](https://hub.docker.com/) when using helm installation you can use
-`sumologic.pullSecrets` to pass Kubernetes secret names that contain
-the required credentials.
+In order to authenticate with AWS Public ECR (to prevent hitting unauthenticated
+pulls quota) when using helm installation one can use `sumologic.pullSecrets`
+to pass Kubernetes secret names that contain the required credentials.
 
-Creating aforementioned secret is beyond the scope of this document.
-Extensive documentation on this subject can be found at
-[Creating a Secret with a Docker config at kubernetes.io][k8s-docker-secret].
+Secret with registry credentials can be created using the following commands:
 
-[docker-rate-limit]: https://www.docker.com/increase-rate-limits
+```
+kubectl create secret docker-registry ${SECRET_NAME} \
+  --docker-server=public.ecr.aws \
+  --docker-username=AWS \
+  --docker-password=$(aws ecr-public --region us-east-1 get-login-password)
+```
+
+After creating the secret one can use it in the following way:
+
+```yaml
+sumologic:
+  ...
+  pullSecrets:
+    - name: ${SECRET_NAME}
+```
+
+For more information on using Kubernetes secrets with container registries please
+refer to [Creating a Secret with a Docker config at kubernetes.io][k8s-docker-secret].
+
+[aws-public-ecr-docs]: https://aws.amazon.com/blogs/aws/amazon-ecr-public-a-new-public-container-registry/
```

Comment:
This blog article is a very nicely put piece of information, but can it become outdated (as a blog post could)?

There's this page about pricing https://aws.amazon.com/ecr/pricing/, not so clear in my mind, but should we use this instead?

Reply:
Amended. PTAL